src/garage/torch/algos/multi_task_sac.py

```diff
@@ -0,0 +1,246 @@
+"""This modules creates a MTSAC model in PyTorch."""
```

Comment:
if the name of the main class is MTSAC, the name of the module should be mtsac.py